lib/core/src/lib/datatable/components/boolean-cell/boolean-cell.component.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('BooleanCellComponent', () => {
         const booleanCell = getBooleanCell();
 
         expectedOccurrence ? expect(booleanCell).toBeTruthy() : expect(booleanCell).toBeFalsy();
-        if (expectedLabel) {
+        if (expectedLabel || expectedLabel === '') {
```

Comment:
this change is redundant, checking "if(expectedLabel)" already checks for "null", "undefined", and "empty string" in JS

Reply:
so when passing empty string, it will break on first check as `false` and won't check the second, you might want `|` instead of `||`

Reply:
Good point, thx